From 449d9961a37ebb98034f0792d6d377ee3ccb5c65 Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Thu, 25 Dec 2025 18:50:32 +0100
Subject: [PATCH 11/14] HACK: arm64: dts: apple: t60xx: j[34]1[46]: Mark
 ps_atc1_common as always on

Works around missing suspend/resume handling in ATC phy for DP-altmode.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi b/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi
index 09ca581e6..216d74e90 100644
--- a/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi
+++ b/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi
@@ -295,6 +295,10 @@ &dpaudio2 {
 &atcphy1_xbar {
         status = "okay";
 };
+
+&ps_atc1_common {
+	apple,always-on; /* Needs to stay on for DP-alt suspend/resume */
+};
 #endif
 
 /* Virtual regulator representing the shared shutdown GPIO */
-- 
2.54.0

